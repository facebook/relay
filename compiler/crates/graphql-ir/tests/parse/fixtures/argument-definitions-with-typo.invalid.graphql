# expected-to-throw
fragment TestFragment on User
  @argumentDefinitions(
    id: {type: "ID"}
    size: {type: "[Int]", defaultValu: [100]} # uh-oh, typo
  ) {
  ...Foo @arguments(size: $size)
}
