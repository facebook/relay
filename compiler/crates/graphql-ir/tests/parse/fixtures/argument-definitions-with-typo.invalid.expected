==================================== INPUT ====================================
# expected-to-throw
fragment TestFragment on User
  @argumentDefinitions(
    id: {type: "ID"}
    size: {type: "[Int]", defaultValu: [100]} # uh-oh, typo
  ) {
  ...Foo @arguments(size: $size)
}
==================================== ERROR ====================================
Invalid values passed to '@arguments', supported options include 'type' and 'defaultValue', got 'defaultValu':
argument-definitions-with-typo.invalid.graphql:4:10:
    size: {type: "[Int]", defaultValu: [100]} # uh-oh, typo
  ) {
